src/Backoffice/Form/MyAccount/ChangeEmailDataMapper.php

```diff
@@ -52,10 +52,9 @@ public function mapFormsToData(Traversable $forms, &$viewData): void
         }
 
         $forms = iterator_to_array($forms);
-        $emailDomain = '@' . $emailParts[1];
```

Comment:
This file should also remove the `count($emailParts) !== ` check since it was project specific.